play/src/front/Api/Events/Ui/ButtonActionBarEvent.ts

```diff
@@ -1,9 +1,20 @@
 import { z } from "zod";
 
+export const isAddActionBarButtonEvent = z.object({
+    id: z.string(),
+    label: z.string().optional(),
+    toolTip: z.string().optional(),
+    bgColor: z.string().optional(),
+    textColor: z.string().optional(),
+    imageSrc: z.string().optional(),
+    isGradient: z.boolean().optional().default(false),
+});
+
 export const isAddClassicButtonActionBarEvent = z.object({
     id: z.string(),
     label: z.string(),
-    type: z.enum(["button"]).optional().default("button"),
+    bgColor: z.string().optional(),
+    textColor: z.string().optional(),
```

Comment:
`isAddClassicButtonActionBarEvent` is now a subtype of `isAddActionBarButtonEvent`.
So we can remove this type completely